mrp_subcontracting_partner_management/models/res_partner.py

```diff
@@ -6,258 +8,219 @@ class ResPartner(models.Model):
 
     is_subcontractor_partner = fields.Boolean(string="Subcontractor")
     subcontracted_created_location_id = fields.Many2one(
-        copy=False, comodel_name="stock.location"
+        comodel_name="stock.location", copy=False
     )
     partner_picking_type_id = fields.Many2one(
-        copy=False, comodel_name="stock.picking.type"
-    )
-    partner_buy_rule_id = fields.Many2one(
-        copy=False,
-        comodel_name="stock.rule",
-    )
-    partner_resupply_rule_id = fields.Many2one(
-        copy=False,
-        comodel_name="stock.rule",
+        comodel_name="stock.picking.type", copy=False
     )
+    partner_buy_rule_id = fields.Many2one(comodel_name="stock.rule", copy=False)
+    partner_resupply_rule_id = fields.Many2one(comodel_name="stock.rule", copy=False)
+
+    CHECK_DATA = {
```

Comment:
A good practice is that this is returned by a hook that can be inherited.

Reply:
@pedrobaeza What do you mean?

Reply:
Resolved. Wrapped CHECK_DATA in function.